tests/test_baseclass.py

```diff
@@ -431,12 +431,40 @@ def load_prediction_data(self, *args, **kwargs):
 
 
 class TestScoreEstimator:
-    def test_score_estimator_fails_if_no_train_test_data_available(self, iris_dataset):
+    def test_score_estimator_if_no_train_test_data_available(
+        self, boston_dataset, train_boston_dataset
+    ):
         model = Model(LinearRegression())
+        data = boston_dataset()
+
+        test = train_boston_dataset
+        result = model.score_estimator(test)
+
+        assert model.score_estimator(data) == result
+
+    def test_score_estimator_if_no_train_test_data_available_classification(
+        self, iris_dataset, train_iris_dataset
+    ):
+        model = Model(LogisticRegression())
         data = iris_dataset()
 
-        with pytest.raises(MLToolingError, match="Must run create_train_test first!"):
-            model.score_estimator(data)
+        test = train_iris_dataset
+        result = model.score_estimator(test)
+
+        assert model.score_estimator(data) == result
+
+    def test_score_estimator_if_no_train_test_data_available_changed_config(
+        self, iris_dataset, train_iris_dataset
```

Comment:
Looks like `train_iris_dataset` is not being used?

What is this testing?

Reply:
`train_iris_dataset` should not be included and has been removed.

This test checks  if the `score_estimator` correctly calculates the score estimator without any test data has been create and the config has been changed
```suggestion
        def test_score_estimator_creates_train_test_data_with_changed_config(
        self, iris_dataset
```